demo/package.json

```diff
@@ -17,6 +17,7 @@
     "prop-types": "^15.6.2",
     "react": "16.5.0",
     "react-native": "https://github.com/expo/react-native/archive/sdk-32.0.0.tar.gz",
+    "react-native-button": "^2.3.0",
```

Comment:
For a component this small that may not be maintained it’s also tempting to incorporate the code directly in the codebase. But I like using it as a package here, at least for now